stdlib/LinearAlgebra/src/tridiag.jl

```diff
@@ -604,6 +617,19 @@ function diag(M::Tridiagonal{T}, n::Integer=0) where T
     end
 end
 
+@inline function Base.isassigned(A::Tridiagonal, i::Int, j::Int)
+    @boundscheck checkbounds(A, i, j)
```

Comment:
is this ok? in all other `isassigned` implementations, this returns false if the bound check fails, but this throws an error?

Reply:
You're right. That should be fixed